concepts/rules.md

```diff
@@ -21,6 +21,14 @@ openHAB can send a notification to your phone.
 Other systems may have a concept of _Automations_, _Tasks_, and other terms.
 In openHAB, rules are used to implement all of these concepts.
 
+Rules can be executed, either when fired by their triggers, or when called explicitly - from a script, another rule, MainUI widget.
+When a rule is fired by its trigger, the execution of one and the same rule does not happen in parallel.
+If a rule is triggered for execution, while the rule is currently running, it will be queued and run later.
+
+When a rule is called explicitly - from a script, another rule with [`RuleManager.runNow()`](https://www.openhab.org/javadoc/latest/org/openhab/core/automation/rulemanager), from a MainUI widget, then this rule can be executed in parallel.
```

Comment:
```suggestion
When a rule is called explicitly - from a script, another rule with [`RuleManager.runNow()`](https://www.openhab.org/javadoc/latest/org/openhab/core/automation/rulemanager), or from a Main UI widget, then this rule can be executed in parallel.
```

Reply:
HTTP POST `/rest/rules/{rule-uid}/runnow/` also executes the rule extraordinary.  The lack of “or” is supposed to indicate that the list is not exclusive.